src/ripple/app/ledger/LedgerMaster.h

```diff
@@ -371,6 +371,35 @@ class LedgerMaster
     // without first wiping the database.
     LedgerIndex const max_ledger_difference_ {1000000};
 
+private:
+    struct Stats
+    {
+        template <class Handler>
+        Stats (Handler const& handler, beast::insight::Collector::ptr const& collector)
+            : hook (collector->make_hook (handler)),
+            mValidatedLedgerAge (collector->make_gauge ("LedgerMaster", "Validated_Ledger_Age")),
```

Comment:
Formatting micro-nit:
```
    template <class Handler>
    Stats (Handler const& handler, beast::insight::Collector::ptr const& collector)
        : hook (collector->make_hook (handler))
        , validatedLedgerAge (collector->make_gauge ("LedgerMaster", "Validated_Ledger_Age"))
        , publishedLedgerAge (collector->make_gauge ("LedgerMaster", "Published_Ledger_Age"))
    { }
```

Also, note how I avoided the `m` prefix by using just `validatedLedgerAge` and `publishedLedgerAge`.

Reply:
Thanks! I fixed it.